packages/components/atoms/f-form-field/stories/FormField.stories.js

```diff
@@ -81,10 +80,8 @@ export const FormFieldComponent = () => ({
             :rows="7"
             :maxlength="200"
             :labelDetails="labelDetails"
-            :assistiveText="assistiveText"
-            :has-input-description="hasInputDescription">
-                Here is a bit more text to show
-        </form-field>`
+            :labelDescription="labelDescription"
```

Comment:
Same as above – are the current tests sufficient for the new prop rather than what was testing the slot previously?